evaluation/swe_bench/run_infer.py

```diff
@@ -269,18 +276,21 @@ def complete_runtime(
     workspace_dir_name = _get_swebench_workspace_dir_name(instance)
 
     action = CmdRunAction(command=f'cd /workspace/{workspace_dir_name}')
+    action.timeout = 600
     logger.info(action, extra={'msg_type': 'ACTION'})
     obs = runtime.run_action(action)
     logger.info(obs, extra={'msg_type': 'OBSERVATION'})
     assert obs.exit_code == 0
 
     action = CmdRunAction(command='git config --global core.pager ""')
+    action.timeout = 600
     logger.info(action, extra={'msg_type': 'ACTION'})
     obs = runtime.run_action(action)
     logger.info(obs, extra={'msg_type': 'OBSERVATION'})
     assert obs.exit_code == 0
 
     action = CmdRunAction(command='git add -A')
+    action.timeout = 600
```

Comment:
Worth trying, thank you. There is a new issue reported which fails on one of these initial commands, maybe it will address it.

Reply:
These changes are comes from https://github.com/All-Hands-AI/OpenHands/pull/3829 😄. Would appreciate if you could take a look there and see if we can get that merged (pre-req before this PR).